MOVIShield.cpp

```diff
@@ -45,11 +45,13 @@
 
 MOVI::MOVI()
 {
+    usehardwareserial=false;
     construct(ARDUINO_RX_PIN, ARDUINO_TX_PIN, false);
 }
 
-MOVI::MOVI(bool debugonoff)
+MOVI::MOVI(uint8_t debugonoff)
```

Comment:
That makes sense. It can be solved by requiring the debug parameter with the constructor that takes the hardwareserial. This makes it more consistent too:
MOVI(bool debugonoff, int rx, int tx)
MOVI(bool debugonoff, HardwareSerial *hs)